Makefile

```diff
@@ -28,6 +28,7 @@ export PATH := $(shell go env GOPATH)/bin:$(PATH)
 export PATH := ./bin:$(PATH)
 export TERM := linux-m
 export GO111MODULE := on
+export GOTOOLCHAIN := local
```

Comment:
I found the "auto" behavior with asdf to be dangerous as it installs versions within the version manager so avoiding issue shere